test/testdata/lsp/fast_path/static_field_type.2.rbupdate

```diff
@@ -0,0 +1,10 @@
+# typed: true
+# assert-fast-path: static_field_type.rb
+
+class A
+  def foo(x); end
+
+  X = ''
+end
+
+T.reveal_type(A::X) # error: `String`
```

Comment:
ooc, are these `error:` assertions checked in `*.rbupdate` files? if this said the wrong type would the test fail?

Reply:
They are checked!  Makes so many things easy.